music21/stream/base.py

```diff
@@ -6432,6 +6432,9 @@ def makeNotation(self,
         measureStream = returnStream.getElementsByClass('Measure').stream()
         # environLocal.printDebug(['Stream.makeNotation(): post makeMeasures,
         #   length', len(returnStream)])
+        if not measureStream:
+            raise StreamException(
+                f'no measures found in stream with {len(self)} elements')
```

Comment:
Can you explain this change?  I'm neither for or against it, just not sure what the utility is?  Does stream.Stream().show() still work after this?